### ISSUE TITLE
add MariadbGTIDEvent.Flags

### ISSUE THREAD

**Issue by gleonid** [siddontang/go-mysql#418]
**Feature request**:
surface Flags of MariadbGTIDEvent

**Background**
According to MariaDB docs: https://mariadb.com/kb/en/library/gtid_event/ 
GTID event has a Flags bitmask that has info about transaction.
We are specifically looking for FL_STANDALONE flag. Knowing whether transaction is "standalone" or not will allow our application to correctly anticipate events sequence for transaction. We are processing row based replication stream.

In other words - we need to know which event is the last event in the transaction.
if FL_STANDALONE is set - query event completes transaction
if FL_STANDALONE is not set - xid event completes transaction
